cmd/podman/manifest/push.go

```diff
@@ -120,6 +122,16 @@ func init() {
 }
 
 func push(cmd *cobra.Command, args []string) error {
+	podmanConfig, err := config.ReadCustomConfig()
```

Comment:
You need to use `podmanConfig := registry.PodmanConfig()` and use `podmanConfig.ContainersConfDefaultsRO.Engine`.  There are many users of that in `cmd/podman` if you want to have a look.

Reply:
Can we set the value directly as the default of the CLI flag?